backend/tests/unit/test_user.py

```diff
@@ -0,0 +1,127 @@
+import pytest
```

Comment:
```
tests/unit/test_csv.py F......                                                                [ 87%]
tests/unit/test_user.py s                                                                     [100%]
============================================= FAILURES ==============================================
____________________________________________ test_basic _____________________________________________
    def test_basic():
        result = generate_csv_from_list(person)
>       assert (
            result
            == “Person1,20,\“[{‘name’: ‘Beans’, ‘type’: ‘Cat’}, ”
            + “{‘name’: ‘Spot’, ‘type’: ‘Dog’}]\“,2\r\nPerson2,25,\“”
            + “[{‘name’: ‘Splash’, ‘type’: ‘Fish’}]\“,1\r\n”
        )
E       assert ‘Person1,20,“...Fish\‘}]“\r\n’ == ‘Person1,20,“...sh\‘}]“,1\r\n’
E
E         Skipping 70 identical leading characters in diff, use -v to show
E         - : ‘Dog’}]“,2
E         ?           --
E         + : ‘Dog’}]”
E         - Person2,25,“[{‘name’: ‘Splash’, ‘type’: ‘Fish’}]“,1
E         ?                                                  --
E         + Person2,25,“[{‘name’: ‘Splash’, ‘type’: ‘Fish’}]”
tests/unit/test_csv.py:112: AssertionError
```

Got the following error after running `pdm run pytest`, let me know if you can replicate it.

Reply:
Feel free to delete this, looks like it's from `test_csv.py`